docs/dev.md

```diff
@@ -442,7 +442,7 @@ be done on the CI in the future.
 It is possible to test push notifications with the `--push` flag. You need first to
 configure the stack so that it uses a fake server that will receive push notifications.
 
-- Add this to `~/.cozy.yml`:
+- Add this to `~/cozy.yml`:
```

Comment:
I think the stack config file can vary. For example in [this piece of doc](https://docs.cozy.io/en/howTos/dev/services/#copy-the-default-config-file), it's under `~/.cozy/cozy.yaml` (that's what I personally use). So I think we should not give a specific path here, but just say "Add this to your cozy-stack configuration file".

Reply:
Correct, I fixed and added a link to the doc.